renovate.json

```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["local>freckle/renovate-config"],
```

Comment:
I know you just got this from somewhere, but I'm confused to see `local>` here. You'd think this is very much a remote thing being reference. :thinking: Oh well.